torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -183,12 +183,8 @@ def clamp_bounding_box(
     return convert_format_bounding_box(xyxy_boxes, BoundingBoxFormat.XYXY, format)
 
 
-def _split_alpha(image: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
-    return image[..., :-1, :, :], image[..., -1:, :, :]
```

Comment:
`tensor_split` is marginally faster because it avoids the double indexing:
```
[------------------------------- Convert_color_space_image_tensor cpu torch.uint8 ------------------------------]
                     |  convert_color_space_image_tensor v2 before  |  convert_color_space_image_tensor_ v2 after
1 threads: ------------------------------------------------------------------------------------------------------
      (4, 400, 400)  |                     217                      |                     213                    
6 threads: ------------------------------------------------------------------------------------------------------
      (4, 400, 400)  |                     297                      |                     292                    

Times are in microseconds (us).

[------------------------------ Convert_color_space_image_tensor cuda torch.uint8 ------------------------------]
                     |  convert_color_space_image_tensor v2 before  |  convert_color_space_image_tensor_ v2 after
1 threads: ------------------------------------------------------------------------------------------------------
      (4, 400, 400)  |                     56.4                     |                      52                    
6 threads: ------------------------------------------------------------------------------------------------------
      (4, 400, 400)  |                     56.4                     |                      52                    

Times are in microseconds (us).

[------------------------------ Convert_color_space_image_tensor cpu torch.float32 -----------------------------]
                     |  convert_color_space_image_tensor v2 before  |  convert_color_space_image_tensor_ v2 after
1 threads: ------------------------------------------------------------------------------------------------------
      (4, 400, 400)  |                     236                      |                     232                    
6 threads: ------------------------------------------------------------------------------------------------------
      (4, 400, 400)  |                     325                      |                     321                    

Times are in microseconds (us).

[----------------------------- Convert_color_space_image_tensor cuda torch.float32 -----------------------------]
                     |  convert_color_space_image_tensor v2 before  |  convert_color_space_image_tensor_ v2 after
1 threads: ------------------------------------------------------------------------------------------------------
      (4, 400, 400)  |                     56.9                     |                     52.4                   
6 threads: ------------------------------------------------------------------------------------------------------
      (4, 400, 400)  |                     56.7                     |                     52.4                   

Times are in microseconds (us).
```